rails-tasks/app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="on_paper">
+<h1>editing <%= @task.name %></h1>
```

Comment:
This partial for the form is going to be used in multiple templates. Might it be better to have the `div` and `h1` tags specified in the containing template, as not all of those cases involves editing an existing Task object.